docs/ref/urlresolvers.txt

```diff
@@ -130,6 +130,12 @@ If the URL does not resolve, the function raises a
 
         The name of the URL pattern that matches the URL.
 
+    .. attribute:: ResolverMatch.route
+
+        .. versionadded:: 2.1
```

Comment:
I believe this should be 2.2?

Reply:
Yes, and please add a release note in `docs/releases/2.2.txt`.